dev/index.js

```diff
@@ -1,6 +1,6 @@
 import React, {Component} from 'react';
 import ReactDOM from 'react-dom';
-import plotly from 'plotly.js/dist/plotly-basic';
+import plotly from 'plotly.js/dist/plotly';
```

Comment:
Hmmm this might cause an issue with webpack when uglifying... The test I'm running now is almost at 10 minutes and it's still not done :(

Reply:
`-basic` took 153s to complete with `webpack -p`

Reply:
but this is what I need to get all my other chart types..
if I don't have this, I only get the basic 5 chart types, I think it's ok that we add a fuller version into the dev env

Reply:
for the dev env ok but this won't work for something like the simple app: we'll need to load plotly.js the proper way i.e. not from `dist` :)

Reply:
ah I see : ) ok, well, I'll merge this for now